pyglmnet/pyglmnet.py

```diff
@@ -175,9 +175,14 @@ def _grad_L2loss(distr, alpha, Tau, reg_lambda, X, y, eta, beta):
                    ((1 - y) * (grad_prob / (1 - prob))))
         grad_logl = ((y * (grad_prob / prob)) -
                      ((1 - y) * (grad_prob / (1 - prob))))
-        grad_beta = -1. / n_samples * np.transpose(np.dot(grad_logl.T, X)) + \
-            reg_lambda * (1 - alpha) * np.dot(InvCov, beta[1:])
+        grad_beta = -1. / n_samples * np.transpose(np.dot(grad_logl.T, X))
 
+    elif distr == 'gamma':
+        nu = 1.
+        grad_beta0 = 1. / n_samples * nu * np.sum(-y + 1 / z)
+        grad_beta = 1. / n_samples * nu * (-np.dot(y, X) + np.dot(1 / z, X))
+
+    grad_beta += reg_lambda * (1 - alpha) * np.dot(InvCov, beta[1:])
```

Comment:
I just moved this term out of the if statements because it was occurring in all of them